src/NATS.Client/JetStream/JetStreamMsg.cs

```diff
@@ -0,0 +1,242 @@
+﻿// Copyright 2021 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Text;
+
+namespace NATS.Client.JetStream
+{
+    public sealed class JetStreamMsg : Msg
+    {
+        static readonly private byte[] ackAck = Encoding.ASCII.GetBytes("+ACK");
+	    static readonly private byte[] ackNak = Encoding.ASCII.GetBytes("-NAK");
+	    static readonly private byte[] ackProgress = Encoding.ASCII.GetBytes("+WPI");
+	    static readonly private byte[] ackTerm = Encoding.ASCII.GetBytes("+TERM");
+
+        /// <summary>
+        /// Gets the metadata associated with a JetStream message.
+        /// </summary>
+        public override MetaData Metadata { get; }
```

Comment:
If the class is `MetaData` then I suggest the property matches:
```suggestion
        public override MetaData MetaData { get; }
```

Reply:
fixed